scripts/backup.sh

```
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -eu
+
+NETWORK="mainnet"
+BASE_DIR="${HOME}/snarkOS/aleo_ledger_checkpoints/${NETWORK}/"
+ENDPOINT="http://localhost:3030/${NETWORK}/db_backup"
+JWT="eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhbGVvMXJoZ2R1NzdoZ3lxZDN4amo4dWN1M2pqOXIya3J3ejZtbnp5ZDgwZ25jcjVmeGN3bGg1cnN2enA5cHgiLCJpYXQiOjE3NDkxMTYzNDUsImV4cCI6MjA2NDQ3NjM0NX0.LiqFGiQds3OGHGJ5K3xi359g-uTQBZCrAskGj9UWAbM"
+
+post_backup() {
+    slot="$1"
+    slot_path="${BASE_DIR}/${slot}"
+    rm -rf -- "$slot_path" 2>/dev/null || :
+    curl -fsS -X POST -H "Authorization: Bearer ${JWT}" "${ENDPOINT}?path=${slot_path}" >/dev/null 2>&1 || :
```

Is it a good idea to discard the return value here? IMHO it should print something when it fails or successfully creates a backup.

---

The script is just an example and likely not something directly usable in production, but good point, especially for testing it should not discard errors: https://github.com/ProvableHQ/snarkOS/commit/be226d9c59a3d1aa9f979a5e8fcc1767f1e5d817